publish.go

```diff
@@ -0,0 +1,127 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/charmbracelet/keygen"
+	gap "github.com/muesli/go-app-paths"
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+)
+
+const (
+	ghostHost = "ghost.charm.sh"
+	ghostPort = 22
+)
+
+var publishCmd = &cobra.Command{
+	Use:           "publish <gif>",
+	Short:         "Publish a GIF to the web",
+	Args:          cobra.ExactArgs(1),
+	SilenceUsage:  true,
+	SilenceErrors: true, // we print our own errors
+	RunE: func(cmd *cobra.Command, args []string) error {
+		url, err := Publish(cmd.Context(), args[0])
+		if err != nil {
+			return err
+		}
+		fmt.Println(StringStyle.Render("URL: " + url))
+		return nil
+	},
+}
+
+func dataPath() (string, error) {
+	scope := gap.NewScope(gap.User, "vhs")
+	dataPath, err := scope.DataPath("")
+	if err != nil {
+		return "", err
+	}
+	return dataPath, nil
+}
+
+func sshSession() (*ssh.Session, error) {
+	dp, err := dataPath()
+	if err != nil {
+		return nil, err
+	}
+	kp, err := keygen.NewWithWrite(filepath.Join(dp, "vhs"), nil, keygen.Ed25519)
+	if err != nil {
+		return nil, err
+	}
+
+	signer, err := ssh.NewSignerFromKey(kp.PrivateKey())
+	if err != nil {
+		return nil, err
+	}
+
+	pkam := ssh.PublicKeys(signer)
+	sshConfig := &ssh.ClientConfig{
+		User:            "vhs",
+		Auth:            []ssh.AuthMethod{pkam},
+		HostKeyCallback: ssh.InsecureIgnoreHostKey(), //nolint
```

Comment:
```suggestion
		HostKeyCallback: ssh.InsecureIgnoreHostKey(),
```

Reply:
could probably be copied from: https://github.com/charmbracelet/wishlist/blob/43ce3d6a27c6e54a880b643d4de9f8ac7f578a85/client_auth.go#L254